scripts/upgrade

```diff
@@ -71,19 +71,19 @@ fi
 #=================================================
 
 [[ $YNH_APP_ID == "garradin" ]] \
-    && [[ "$(cat "/opt/$app/templates/.VERSION")" != 1.2.2 ]] \
+    && [[ "$(cat "$final_path/VERSION")" != 1.2.2 ]] \
     && ynh_die --message "It look like that you have an old Garradin install. You need first upgrade Garradin instance (id: $garradin_migrate_id) and after migrate to Paheko."
 ynh_handle_app_migration --migration_id=garradin --migration_list=garradin_migrations
 
 if [[ $migration_process -eq 1 ]]; then
     # Reload variables
-    dbname=$app
-    db_user=$app
+    #dbname=$app
+    #db_user=$app
 
     # Replace the user
     ynh_system_user_delete --username="$old_app"
-    test getent passwd "$app" &>/dev/null || \
-        useradd -d "$datadir" --system --user-group "$app" --shell /bin/bash || \
+    #test getent passwd "$app" &>/dev/null || \
+        useradd -d "$final_path" --system --user-group "$app" || \
             ynh_die --message "Unable to create $app system account"
 
```

Comment:
I guess you forgot to put a `fi` here to clause the `if` block here.

Reply:
I am not sure, I think the `fi` is the end of the `experimental_helper.sh`

Reply:
non c'est pas ça... perhaps you are right.

Reply:
You are right and I have forgotten add a file garradin_migration !!